src/test/locale/es-do.js

```diff
@@ -211,3 +211,9 @@ test('weeks year starting sunday formatted', function (assert) {
     assert.equal(moment([2012, 0, 15]).format('w ww wo'),   '2 02 2º', 'Jan 15 2012 should be week 2');
 });
 
+test('test short months proper', function (assert) {
+    var str = moment(new Date(2016, 7, 2)).format('DD-MMM-YYYY'); // "02-ago-2016"
```

Comment:
Not really, if the formatting breaks it will be the last test to fail so it's pretty much redundant. It was copied from the recreation steps of the issue reporter, i'll go ahead and replace it with the string